docs/sphinxdocs/source/conf.py

```diff
@@ -48,7 +48,7 @@
 
 autodoc_default_flags = ["members"]
 autodoc_member_order = "bysource"
-autosummary_generate = True
+autosummary_generate = False
```

Comment:
Thanks for getting me to double-check. There actually were side effects which I hadn't thoroughly investigated. These stub files do need to be generated so that sphinx explores secondary links (e.g. it would still generate the top-level summaries of `Arctic` etc, but not to the lower level ones without the stubs).

Solution was to set this back to `True`, and to add all the stubs to `.gitignore`.